submissions/VikTorNik/html-css-popup/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hooli-style Popup</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <nav class="nav">
+                <ul class="nav__menu">
+                    <li class="menu-button"> <a class="menu__item" href="#">
```

Comment:
Avoid shifting sibling elements on hover or focus state.
[scrnli_10_1_2022_12-08-30 PM.webm](https://user-images.githubusercontent.com/16196199/193402332-cb488c8a-b7e3-4fc7-8930-0cb28d171630.webm)

Reply:
Fixed it.